frameworks/Java/redkale/src/main/java/org/redkalex/benchmark/Service.java

```diff
@@ -88,7 +90,7 @@ public Message getHelloMessage() {
             }
         }
         final int size = Math.min(500, Math.max(1, q));
-        return cache.random(ThreadLocalRandom.current(), size);
+        return cache.random(cached ? rands.get() : ThreadLocalRandom.current(), size);
```

Comment:
Random numbers should never be cached? What is this doing?

Reply:
/cached-worlds  use RedRandom.class   other use ThreadLocalRandom
look at the performance difference between the two

Reply:
/cached-worlds use RedRandom.class other use ThreadLocalRandom
look at the performance difference between the two